baggers.js

```diff
@@ -8459,26 +8459,11 @@ var data = {
         },
         {
           name: 'Web',
-          url: 'http://adrienjoly.com/',
+          url: 'https://adrienjoly.com/',
         },
       ],
       location: 'Montreuil / Paris',
-      sessions: [
-        {
-          title: 'Comment devenir CTO, en évitant les erreurs de débutant',
-          abstract:
-            "Après 4 ans comme lead developer dans une startup web parisienne, je propose de venir partager un retour d'expérience sur cette aventure, et des conseils pratiques sur: comment reconnaitre un bon associé d'un incompétent, comment éviter de développer un service voué à l'échec, comment obtenir la confiance de son associé pour les choix technologiques et les délais de développement.",
-          tags: [
-            'Javascript',
-            'Lean Startup',
-            'Bootstrapping',
-            'CEO',
-            'CTO',
-            'Trello',
-          ],
-          lang: ['fr'],
-        },
-      ],
+      sessions: [],
```

Comment:
But if you don't have sessions anymore you should better remove your profile no?

Reply:
That's a fair point! => I added 3 sessions.